virttest/test_setup.py

```diff
@@ -2537,6 +2539,7 @@ def enable(self):
             value.get("conf")["log_level"] = self.log_level
             value.get("conf")["log_outputs"] = '"%s:file:%s"' % (self.log_level,
                                                                  self.log_file)
+            value.get("conf")["log_filters"] = '"%s"' % self.log_filters
```

Comment:
@Yingshun, here is the example in the `virtlogd.conf`: `#log_filters="1:logging 4:object 4:json 4:event 1:util"`. So if we define multiple filters, we should use quotes to bind them. Without quotes, `virtlogd` services failed to restart.
```
# grep -P '^\w' /etc/libvirt/virtlogd.conf
log_filters=1:logging 4:object 4:json 4:event 1:util
# systemctl restart virtlogd
# systemctl status virtlogd
virtlogd.service - Virtual machine log manager
     Loaded: loaded (/usr/lib/systemd/system/virtlogd.service; indirect; vendor preset: disabled)
     Active: failed (Result: exit-code) since Tue 2022-05-31 23:53:32 EDT; 6s ago
   Duration: 22ms
TriggeredBy: ● virtlogd-admin.socket
             ● virtlogd.socket
       Docs: man:virtlogd(8)
             https://libvirt.org
    Process: 24231 ExecStart=/usr/sbin/virtlogd $VIRTLOGD_ARGS (code=exited, status=1/FAILURE)
   Main PID: 24231 (code=exited, status=1/FAILURE)
        CPU: 22ms

May 31 23:53:32 virtlab-arm02.virt.lab.eng.bos.redhat.com systemd[1]: Started Virtual machine log manager.
May 31 23:53:32 virtlab-arm02.virt.lab.eng.bos.redhat.com virtlogd[24231]: 2022-06-01 03:53:32.163+0000: 24231: info : libvirt version: 8.3.0, package: 1.el9 (Red Hat, Inc. <http://bugzilla.redhat.com/bugzilla>, 2022-05-06-05:56:26, )
May 31 23:53:32 virtlab-arm02.virt.lab.eng.bos.redhat.com virtlogd[24231]: 2022-06-01 03:53:32.163+0000: 24231: info : hostname: virtlab-arm02.virt.lab.eng.bos.redhat.com
May 31 23:53:32 virtlab-arm02.virt.lab.eng.bos.redhat.com virtlogd[24231]: 2022-06-01 03:53:32.163+0000: 24231: error : main:715 : Can't load config file: configuration file syntax error: /etc/libvirt/virtlogd.conf:57: expecting a separator: /etc/libvirt/virtlogd.conf
May 31 23:53:32 virtlab-arm02.virt.lab.eng.bos.redhat.com systemd[1]: virtlogd.service: Main process exited, code=exited, status=1/FAILURE
May 31 23:53:32 virtlab-arm02.virt.lab.eng.bos.redhat.com systemd[1]: virtlogd.service: Failed with result 'exit-code'.
```

Reply:
Oops, got it. Thanks for your explanation!